distribution/server/src/assemble/LICENSE.bin.txt

```diff
@@ -494,9 +494,9 @@ The Apache Software License, Version 2.0
     - joda-time-2.10.1.jar
     - joda-time-joda-time-2.10.1.jar
   * Dropwizard
-    - io.dropwizard.metrics-metrics-core-3.1.0.jar
-    - io.dropwizard.metrics-metrics-graphite-3.1.0.jar
-    - io.dropwizard.metrics-metrics-jvm-3.1.0.jar
+    - io.dropwizard.metrics-metrics-core-3.2.5.jar
```

Comment:
Bookkeeper uses Zookeeper 3.6.2 and dropwizard 3.1.0, I think it makes sense to keep dependency versions in sync to avoid surprises later

Reply:
@dlg99 I can't remember why I had to make this bump.
Probably it is the same version as in ZK.

I would prefer to upgrade this dep in BK 4.13.